src/components/Display.js

```diff
@@ -7,7 +7,7 @@ const Display = (props) => {
 };
 
 Display.defaultProps = {
-  result: 'o',
+  result: 0,
 };
 
 Display.propTypes = {
```

Comment:
this should be `result: '0'` 
Given that the propType is string ....
Please rectify this